test/IntegrationTests/test_addcustomer.py

```diff
@@ -1,13 +1,19 @@
 import unittest
 import requests
 import logging
+import os
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+dotenv_path = os.path.join(os.path.dirname(__file__), '..', '..', 'src', 'backend', '.env')
+load_dotenv(dotenv_path)
```

Comment:
Remove these lines, the code will work without these lines

Make sure you remove these lines in all the files

Reply:
I removed those three lines from all files